packages/api-client-core/src/support.ts

```diff
@@ -316,8 +316,12 @@ export const getNonNullableError = (response: Result & { fetching: boolean }, da
 
 export const assertOperationSuccess = (response: OperationResult<any>, dataPath: string[], throwOnEmptyData = false) => {
   if (response.error) {
-    if (response.error instanceof CombinedError && (response.error.networkError as any as Error[])?.length) {
-      response.error.message = (response.error.networkError as any as Error[]).map((error) => "[Network] " + error.message).join("\n");
+    if (response.error instanceof CombinedError && response.error.networkError) {
+      if (response.error.networkError?.message) {
+        response.error.message = `[Network] ${response.error.networkError.message}`;
+      } else {
+        response.error.message = `[Network] No message, stack: ${String(response.error.networkError.stack)}`;
```

Comment:
I think we should stringify the whole error error here instead of relying on just the stack. In JS you can throw anything, like `throw "foo"`, so the error may not have a message or a stack and just be a string. Since we dont exactly know what it is I think it might be wise to be defensive and just stringify the result right out

Reply:
Ah kk, my test was just returning `Error` with no stack when I did `string(response.networkError)` which is why I explicitly grabbed the stack. Would something like
`[Network] No message, error: string(response.error.networkError) \nstack: ${String(response.error.networkError.stack)}` work?

Reply:
yep